06-reactJs/components/form.js

```diff
@@ -0,0 +1,48 @@
+var React = require('react');
+
+var Form = React.createClass({
+    getInitialState: function () {
+        return({
```

Comment:
There shouldn't be parenthesis here, just the curly braces (you're returning an object). Also, remember to add space after ``return``.